pkg/scheduler/scheduler.go

```diff
@@ -291,7 +291,7 @@ func (s *Scheduler) schedule(ctx context.Context) wait.SpeedSignal {
 	for _, e := range entries {
 		logAdmissionAttemptIfVerbose(log, &e)
 		if e.status != assumed {
-			s.requeueAndUpdate(log, ctx, e)
+			s.requeueAndUpdate(ctx, log, e)
```

Comment:
```suggestion
			s.requeueAndUpdate(ctx, e)
```

The log can be obtained from the context within the function

Reply:
Thanks. Changed.